src/RestaurantButton.jsx

```diff
@@ -0,0 +1,23 @@
+import React, { useState } from "react";
+
+const RestaurantButton = props => {
+  const [orders, countOrder] = useState(0);
+  const incrementOrders = () => {
+    countOrder(orders + 1);
+  };
+  return (
+    <div
+      style={{
+        display: "flex",
+        alignContent: "center",
+        justifyContent: "center",
+        flexWrap: "wrap"
+      }}
+    >
+      TotalOrders:{orders}
+      <button onClick={incrementOrders}>Increment Orders</button>
+    </div>
+  );
+};
+
+export default RestaurantButton;
```

Comment:
This is the component I was talking about above ^^ You could import this in your Order component and use it there as the functionality is the same as the buttons there. So you have a little bit of repetition and an extra button at the bottom of your app, but the functionality in both places is good and you are using the hooks correctly and effectively so well done 🥇